

<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="utf-8" />
    <title>RESOLUTE FAIRification Recipe &#8212; The FAIR cookbook</title>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.min.css" integrity="sha384-KA6wR/X5RY4zFAHpv/CnoG2UW1uogYfdnP67Uv7eULvTveboZJg0qUpmJZb5VqzN" crossorigin="anonymous">
    <link href="../../../_static/css/index.css" rel="stylesheet">
    <link rel="stylesheet" href="../../../_static/sphinx-book-theme.css" type="text/css" />
    <link rel="stylesheet" href="../../../_static/pygments.css" type="text/css" />
    <link rel="stylesheet" type="text/css" href="../../../_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="../../../_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="../../../_static/mystnb.css" />
    <link rel="stylesheet" type="text/css" href="../../../_static/jupyter-sphinx.css" />
    <script id="documentation_options" data-url_root="../../../" src="../../../_static/documentation_options.js"></script>
    <script src="../../../_static/sphinx-book-theme.js"></script>
    <script src="../../../_static/jquery.js"></script>
    <script src="../../../_static/underscore.js"></script>
    <script src="../../../_static/doctools.js"></script>
    <script src="../../../_static/language_data.js"></script>
    <script src="../../../_static/togglebutton.js"></script>
    <script src="../../../_static/clipboard.min.js"></script>
    <script src="../../../_static/copybutton.js"></script>
    <script src="../../../_static/mystnb.js"></script>
    <script src="../../../_static/sphinx-book-theme.js"></script>
    <script >var togglebuttonSelector = '.toggle, .admonition.dropdown, .tag_hide_input div.cell_input, .tag_hide-input div.cell_input, .tag_hide_output div.cell_output, .tag_hide-output div.cell_output, .tag_hide_cell.cell, .tag_hide-cell.cell';</script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js"></script>
    <script src="https://unpkg.com/@jupyter-widgets/html-manager@^0.18.0/dist/embed-amd.js"></script>
    <script async="async" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-AMS-MML_HTMLorMML"></script>
    <link rel="canonical" href="https://fairplus.github.io/the-fair-cookbook/content/recipes/resolute_raw/rawRecipe_resolute.html" />
    <link rel="index" title="Index" href="../../../genindex.html" />
    <link rel="search" title="Search" href="../../../search.html" />

    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="docsearch:language" content="en">


<!-- Opengraph tags -->
<meta property="og:url"         content="https://fairplus.github.io/the-fair-cookbook/content/recipes/resolute_raw/rawRecipe_resolute.html" />
<meta property="og:type"        content="article" />
<meta property="og:title"       content="RESOLUTE FAIRification Recipe" />
<meta property="og:description" content="RESOLUTE FAIRification Recipe  Ingredients  Data transfer tools:  SFTP server  OneDrive  Amazon S3 Buckets  Data hosting platform:  raw data: SRA [Transcriptomi" />


<meta name="twitter:card" content="summary">


  </head>
  <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="80">
    

    <div class="container-xl">
      <div class="row">
          
<div class="col-12 col-md-3 bd-sidebar site-navigation show" id="site-navigation">
    
        <div class="navbar-brand-box">
<a class="navbar-brand text-wrap" href="../../../index.html">
  
  
  <h1 class="site-logo" id="site-title">The FAIR cookbook</h1>
  
</a>
</div>

<form class="bd-search d-flex align-items-center" action="../../../search.html" method="get">
  <i class="icon fas fa-search"></i>
  <input type="search" class="form-control" name="q" id="search-input" placeholder="Search this book..." aria-label="Search this book..." autocomplete="off" >
</form>

<nav class="bd-links" id="bd-docs-nav" aria-label="Main navigation">
  
  <ul class="nav sidenav_l1">
  <li class="">
    <a href="../../intro.html">FAIR Cookbook</a>
  </li>
<li class="navbar-special">
<p class="margin-caption">Get started</p>
</li>
  <li class="">
    <a href="../../code_of_conduct.html">FAIR Cookbook Code of Conduct</a>
  </li>
  <li class="">
    <a href="../../license.html">License for this book</a>
  </li>
</ul>
</nav>

 <!-- To handle the deprecated key -->

<div class="navbar_extra_footer">
  Powered by <a href="https://jupyterbook.org">Jupyter Book</a>
</div>

</div>


          


          
<main class="col py-md-3 pl-md-4 bd-content overflow-auto" role="main">
    
    <div class="row topbar fixed-top container-xl">
    <div class="col-12 col-md-3 bd-topbar-whitespace site-navigation show">
    </div>
    <div class="col pl-2 topbar-main">
        
        <button id="navbar-toggler" class="navbar-toggler ml-0" type="button" data-toggle="collapse"
            data-toggle="tooltip" data-placement="bottom" data-target=".site-navigation" aria-controls="navbar-menu"
            aria-expanded="true" aria-label="Toggle navigation" aria-controls="site-navigation"
            title="Toggle navigation" data-toggle="tooltip" data-placement="left">
            <i class="fas fa-bars"></i>
            <i class="fas fa-arrow-left"></i>
            <i class="fas fa-arrow-up"></i>
        </button>
        
        <div class="dropdown-buttons-trigger">
    <button id="dropdown-buttons-trigger" class="btn btn-secondary topbarbtn" aria-label="Download this page"><i
            class="fas fa-download"></i></button>

    
    <div class="dropdown-buttons">
        <!-- ipynb file if we had a myst markdown file -->
        
        <!-- Download raw file -->
        <a class="dropdown-buttons" href="../../../_sources/content/recipes/resolute_raw/rawRecipe_resolute.md"><button type="button"
                class="btn btn-secondary topbarbtn" title="Download source file" data-toggle="tooltip"
                data-placement="left">.md</button></a>
        <!-- Download PDF via print -->
        <button type="button" id="download-print" class="btn btn-secondary topbarbtn" title="Print to PDF"
            onClick="window.print()" data-toggle="tooltip" data-placement="left">.pdf</button>
    </div>
    
</div>
        <!-- Source interaction buttons -->

<div class="dropdown-buttons-trigger">
    <button id="dropdown-buttons-trigger" class="btn btn-secondary topbarbtn"
        aria-label="Connect with source repository"><i class="fab fa-github"></i></button>
    <div class="dropdown-buttons sourcebuttons">
        
        
        <a class="edit-button" href="https://github.com/FAIRplus/the-fair-cookbook/edit/master/book/content/recipes/resolute_raw/rawRecipe_resolute.md"><button
                type="button" class="btn btn-secondary topbarbtn" data-toggle="tooltip" data-placement="left"
                title="Edit this page"><i class="fas fa-pencil-alt"></i>suggest edit</button></a>
    </div>
</div>


        <!-- Full screen (wrap in <a> to have style consistency -->
        <a class="full-screen-button"><button type="button" class="btn btn-secondary topbarbtn" data-toggle="tooltip"
                data-placement="bottom" onclick="toggleFullScreen()" title="Fullscreen mode"><i
                    class="fas fa-expand"></i></button></a>

        <!-- Launch buttons -->

    </div>
    <div class="d-none d-md-block col-md-2 bd-toc show">
<div class="tocsection onthispage pt-5 pb-3">
    <i class="fas fa-list"></i> On this page
</div>

<nav id="bd-toc-nav">
    <ul class="nav section-nav flex-column">
    
        <li class="nav-item toc-entry toc-h2">
            <a href="#ingredients" class="nav-link">Ingredients</a>
        </li>
    
        <li class="nav-item toc-entry toc-h2">
            <a href="#sample-metadata-etl-pipeline" class="nav-link">Sample metadata ETL pipeline</a>
        </li>
    
        <li class="nav-item toc-entry toc-h2">
            <a href="#transcriptomics-data-community-standard" class="nav-link">Transcriptomics data community standard</a>
        </li>
    
        <li class="nav-item toc-entry toc-h2">
            <a href="#fair-assessment" class="nav-link">FAIR assessment</a>
        </li>
    
        <li class="nav-item toc-entry toc-h2">
            <a href="#summary" class="nav-link">Summary</a>
        </li>
    
    </ul>
</nav>



<div class="tocsection editthispage">
    <a href="https://github.com/FAIRplus/the-fair-cookbook/edit/master/book/content/recipes/resolute_raw/rawRecipe_resolute.md">
        <i class="fas fa-pencil-alt"></i> Edit this page
    </a>
</div>

    </div>
</div>
    <div id="main-content" class="row">
        <div class="col-12 col-md-9 pl-md-3 pr-md-0">
        
              <div>
                
  <div class="section" id="resolute-fairification-recipe">
<h1>RESOLUTE FAIRification Recipe<a class="headerlink" href="#resolute-fairification-recipe" title="Permalink to this headline">¶</a></h1>
<div class="section" id="ingredients">
<h2>Ingredients<a class="headerlink" href="#ingredients" title="Permalink to this headline">¶</a></h2>
<ul class="simple">
<li><p>Data transfer tools:</p>
<ul>
<li><p>SFTP server</p></li>
<li><p>OneDrive</p></li>
<li><p>Amazon S3 Buckets</p></li>
</ul>
</li>
<li><p>Data hosting platform:</p>
<ul>
<li><p>raw data: SRA [Transcriptomics]</p></li>
<li><p>ProteomeXchange [Proteomics]</p></li>
<li><p>Samples metadata: BioSamples</p></li>
</ul>
</li>
<li><p>Data standards</p>
<ul>
<li><p>Transcriptomics community standard: <a class="reference external" href="http://fged.org/projects/minseqe/">MINSEQE</a></p></li>
<li><p>Sample metadata schema: <a class="reference external" href="https://github.com/EBIBioSamples/biosamples-v4/blob/dev/webapps/core/src/main/resources/schemas/core/sample.json">BioSamples JSON schema</a></p></li>
<li><p><a class="reference external" href="https://www.ebi.ac.uk/submission/">Data submission portal</a> schema</p></li>
</ul>
</li>
<li><p>FAIR evaluation standard:</p>
<ul>
<li><p><a class="reference external" href="https://github.com/RDA-FAIR/FAIR-data-maturity-model-WG/tree/master/results%20of%20preliminary%20analysis">RDA FAIR indicators</a></p></li>
</ul>
</li>
</ul>
<p>This recipe provides an example of sample metadata extraction and curation practices, suggests a community standard for transcriptomics data and proposes a transcriptomics JSON schema, which allows direct submission of transcriptomics data to the public databases. Different approaches to assess the FAIR level of the dataset is also discussed.</p>
</div>
<div class="section" id="sample-metadata-etl-pipeline">
<h2>Sample metadata ETL pipeline<a class="headerlink" href="#sample-metadata-etl-pipeline" title="Permalink to this headline">¶</a></h2>
<p>The RESOLUTE datasets include 21 samples of seven cell lines. Each cell line has three replicates. The first draft of cell line metadata was first submitted to the NCBI SRA database and stored in the NCBI BioSample database. The raw sample metadata includes cell line name, provider and other administrative information.</p>
<p>To avoid ambiguity and provide detailed cell line information, the RESOLUTE cell lines were linked to cell lines in the Cellosaurus databases by the Cellosaurus accession ID, for example, cell line “HCT 116” was linked to accession “CVCL_0291”. Cellosaurus also provided enriched cell line information, including disease, species of origin, cell line category. Such information was fetched from the Cellosaurus website and mapped to corresponding samples.</p>
<p>Besides the cell line enriched cell line annotation, the RESOLUTE Data Release Plan also specified the cell line provider, culturing conditions and quality control information. Such information is also added to the cell line. License information was also mapped to the metadata.</p>
<p>The sample metadata can be found in the EBI Biosamples database with additional curation. For example, in sample <a class="reference external" href="https://www.ebi.ac.uk/biosamples/samples/SAMN11893688">SAMN11893688</a>, all missing values were omitted. Attribute cell line, organism, sex and tissue were mapped to ontologies in the OLS database. All curations can record can be found <a class="reference external" href="https://www.ebi.ac.uk/biosamples/samples/SAMN11893688/curationlinks">here</a>.</p>
</div>
<div class="section" id="transcriptomics-data-community-standard">
<h2>Transcriptomics data community standard<a class="headerlink" href="#transcriptomics-data-community-standard" title="Permalink to this headline">¶</a></h2>
<p><a class="reference external" href="http://fged.org/projects/minseqe/">Minimum Information about a high-throughput SEQuencing Experiment (MINSEQE)</a> is a recommended community standard of transcriptomics data. It defined five elements of transcriptomics data that shall be harvested, including metadata, sequence read data, final processed data and experimental protocols.</p>
<p>FAIRsharing organization interpreted the MINSEQE guidelines and translated them into a machine-readable <a class="reference external" href="https://github.com/FAIRsharing/mircat/tree/master/minseqe">MINSEQE recommended JSON schema</a>. The FAIRsharing MINSEQE JSON schema provided a framework of transcriptomics data. When it comes to different experiments, the actual data be extended. <a class="reference external" href="https://raw.githubusercontent.com/ebi-ait/FAIRPlus/master/RESOLUTE/transcriptomics/data/RESOLUTE_transcriptomics_metadata_MINSEQE.json">Here</a> is an example of RESOLUTE transcriptomics data following this schema</p>
<p>One limitation of the FAIRsharing MINSEQE schema is that it is incompatible to the submission standards of popular transcriptomics database, like the European Nucleotide Archive(ENA). Transcriptomics data following the FAIRsharing MINSEQE standard must be modified to be submitted.</p>
<p><img alt="Transcriptomics schema" src="../../../_images/image_03.png" /></p>
<p><em>Figure 1: Transcriptomics schema</em></p>
<p>To solve this problem, we proposed a <a class="reference external" href="https://github.com/ebi-ait/FAIRPlus/tree/master/schemas/transcriptomics_schema">transcriptomics schema</a> which follows the EBI Data Submission Portal (DSP) standard, as well as compatible with the MINSEQE guidelines. The new transcriptomics schema is based on the MINSEQE standard, also allows enriched sample, cell lines, project metadata as different building blocks (Figure 1). This schema is compatible with the DSP schema which allows data validated against this schema to be directly submitted to the DSP and distributed to all related databases. <em>(Note: This schema is still actively updated. Feedback welcomed)</em></p>
</div>
<div class="section" id="fair-assessment">
<h2>FAIR assessment<a class="headerlink" href="#fair-assessment" title="Permalink to this headline">¶</a></h2>
<p>The level of “FAIRness” of the RESOLUTE transcriptomics was first determined using the CMMI maturity model described <a class="reference external" href="https://docs.google.com/document/d/1URLfNpBYkCrICpizKZJ7NE29FddNNcoR0T0o_SQza7U/edit#heading=h.w0g0276fq5i6">in this document.</a> The results of this first assessment can be found <a class="reference external" href="https://docs.google.com/document/d/1Q_Su8kY9uNYfCV30jSIoWNdeV8GxA_DTGAcGOSZscQM/edit?usp=sharing">here</a>. In a follow-up meeting, we defined actions that could be taken to increase the level for certain indicators (see <a class="reference external" href="https://docs.google.com/document/d/1yYDcUvyXzYLfq9NZX23tbgIjCSOenSURjcRj61FMdzQ/edit?usp=sharing">this document</a>). These actions were defined within Github.</p>
<p>A <a class="reference external" href="https://docs.google.com/spreadsheets/d/11-jDoMbuxw8Nreurk7yKzk3EHJ54APAQnBl6VTKZPBk/edit#gid=1559176954">new FAIRplus maturity model</a> was set up, based on the RDA indicators. The focus was only on the indicators regarding Findability/Discoverability at that point. Before we could use this model however, the indicators were updated within the RDA initiative. We used their <a class="reference external" href="https://docs.google.com/spreadsheets/d/1mkjElFrTBPBH0QViODexNur0xNGhJqau0zkL4w8RRAw/edit?usp=sharing">new indicators</a>) (for findability, v0.01) and their level definition to determine the levels for both the transcriptomics and proteomics datasets. The result can be found <a class="reference external" href="https://docs.google.com/spreadsheets/d/1abQ5_sOmBWbxAZhQVEUxQ_ybI1yTi0t-tJAVY5J5VY8/edit?usp=sharing">here</a>.</p>
<p>In the meantime, an improved version of the RDA indicators was released (v0.02). In the results document a comparison/mapping is made between v0.02 and v0.01 for possible future reference. <br />
We have updated the scoring for the transcriptomics data to this new version (v0.02).</p>
<ul class="simple">
<li><p>Results of the scoring (to v0.01):</p>
<ul>
<li><p>transcriptomics data set (as in SRA)</p>
<ul>
<li><p>58% for the mandatory indicators</p></li>
<li><p>63% for the recommended indicators</p></li>
<li><p>4 indicators were thought to be not applicable</p></li>
</ul>
</li>
<li><p>proteomics data set (as in OwnCloud)</p>
<ul>
<li><p>42% for the mandatory indicators</p></li>
<li><p>44% for the recommended indicators</p></li>
<li><p>6 indicators were thought to be not applicable</p></li>
</ul>
</li>
</ul>
</li>
</ul>
<p>The FAIRness level of the transcriptomics dataset for the HUH-7 cell line (as published on SRA) was also determined using the <a class="reference external" href="https://fairsharing.github.io/FAIR-Evaluator-FrontEnd/#%21/#%2F%21">FAIR evaluation services</a> (an online evaluator). The result was this dataset only passed for 3 out of the 22 indicators tested. This was mainly due to the limitations on machine readability of metadata and license details within SRA. The complete results can be viewed <a class="reference external" href="https://fairsharing.github.io/FAIR-Evaluator-FrontEnd/#%21/evaluations/170">here</a>.</p>
</div>
<div class="section" id="summary">
<h2>Summary<a class="headerlink" href="#summary" title="Permalink to this headline">¶</a></h2>
<ol class="simple">
<li><p>Cellosaurus accession is recommended as an identifier for cell lines. Cellosaurus also provides enriched information about different cell lines.</p></li>
<li><p>EBI BioSamples database is the master hosting platform for sample metadata, allowing curation and basic ontology annotation.</p></li>
<li><p>MINSEQE is the recommended community standard for transcriptomics data.</p></li>
<li><p>A new transcriptomics schema was proposed which reflects the transcriptomics community guideline and supported by the EBI database submission platform.</p></li>
</ol>
</div>
</div>


              </div>
              
        </div>
    </div>
    
    
    <div class='prev-next-bottom'>
        

    </div>
    <footer class="footer mt-5 mt-md-0">
    <div class="container">
      <p>
        
          By The FAIRplus project members<br/>
        
            &copy; Copyright 2020 (CC-BY 4.0).<br/>
          <div class="extra_footer">
            This project has received funding from the <a href="https://www.imi.europa.eu/">Innovative Medicines Initiative</a> 2 Joint Undertaking under grant agreement No 802750. This Joint Undertaking receives support from the European Union’s Horizon 2020 research and innovation programme and EFPIA Companies. This communication reflects the views of the authors and neither IMI nor the European Union, EFPIA or any Associated Partners are liable for any use that may be made of the information contained herein. <script>mermaid.init();</script>
          </div>
      </p>
    </div>
  </footer>
</main>


      </div>
    </div>

    <script src="../../../_static/js/index.js"></script>
    
    <!-- Google Analytics -->
    <script>
      window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
      ga('create', 'UA-171697285-1', 'auto');
      ga('set', 'anonymizeIp', true);
      ga('send', 'pageview');
    </script>
    <script async src='https://www.google-analytics.com/analytics.js'></script>
    <!-- End Google Analytics -->
    
  </body>
</html>